README.md

```diff
@@ -117,3 +117,23 @@ Then, let it run for a few minutes. Once the operands are deployed, just invoke:
 
 The `-o test-e2e-wait-for-stable-state` skips checking if the operator is deployed. Depending on how do you're testing it,
 you may or may not want to invoke it.
+
+## Using unsupported config overrides options
+
+It is possible (although not supported) to specify custom settings to each Cert Manager image. In order to do it, 
+you need to modify the `co/certmanager` object:
```

Comment:
```
you need to modify the `co/certmanager` object
```
Maybe below is preciser:
``
you need to modify the `certmanager.operator/cluster` object
```

Reply:
:+1: